components/cdc/src/delegate.rs

```diff
@@ -1265,4 +1275,48 @@ mod tests {
             }
         }
     }
+
+    #[test]
+    fn test_is_raw_region() {
+        let region_id = 10;
+        let mut region = Region::default();
+        region.set_id(region_id);
+        region.set_start_key(vec![b'r', 0, 0, 0, b'a']);
```

Comment:
Nit: codes of the 4 cases are duplicated. Suggest to use a vector to organize cases and iterate on it.